app/Models/Category.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    use HasFactory;
+}
```

Comment:
Please add the protected fillable array here with the name attribute. And this is where the eloquent relationships come in. So for instance, the relationship between a category and products is one to many. Look up more about it here:
https://laravel.com/docs/8.x/eloquent-relationships#defining-relationships